ndmg/stats/qa_tensor.py

```diff
@@ -190,7 +184,7 @@ def create_qa_figure(peak_dirs,peak_values,output_dir,model):
             #convert background
             image = np.where(image<=0.01, 255, image)
             #pad image size
-            image = pad_im(image,max_dim,255)
+            image = qa_utils.pad_im(image,max_dim,255,True)
```

Comment:
`255` and `True` should be keyword arguments in this function, right now their meaning is implicit and therefore confusing

Reply:
check commit da08529